dev/is-changed.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import sys
+from argparse import ArgumentParser
+from sparktestsupport.utils import determine_modules_for_files, determine_modules_to_test, identify_changed_files_from_git_commits
```

Comment:
Hm, it actually should complain about the line length .. I actually fixed the linter script to run on dev scripts too before but seems like there's a regression about this. I will take a separate look at this one - there should be more places to fix per the linter not only this script.

Reply:
Got it~ Thanks.

Reply:
Actually, the linter for line-length is disabled for the whole project after https://github.com/apache/spark/pull/34655 .. 😢 . let me make a fix soon after merging this one.